content/blog/2020-08-05-cml-self-hosted-runners-on-demand-with-gpus.md

```diff
@@ -0,0 +1,198 @@
+---
+title: CML self-hosted runners on demand with GPUs
+date: 2020-08-05
+description: |
+  A tutorial on how to setup your own self-hosted runners with GPUs using CML cloud-runners in your CML workflow in Github or Gitlab.
+picture: 2020-08-05/header.png
+author: david_g_ortega
+commentsUrl: https://discuss.dvc.org/t/cml-self-hosted-runners-on-demand-with-gpus/??
+tags:
+  - CML
+  - GPUs
+  - Self-hosted runner
+  - Continuous Machine Learning
+  - Github
+  - Gitlab
+  - Technical debt
+  - Tutorial
+---
+
+# CML self-hosted runners on demand with GPUs
+
+Normally, CI/CD workflows are set up as a collection of jobs that are picked and
```

Comment:
The whole first part is very long and monotonous I would say. At the very least let's add links (like self-hosted runners), use subtitles, simplify a bit, use inline code when it's needed, etc.